site/reference/fn/eval/README.md

```diff
@@ -170,15 +177,34 @@ $ kpt fn eval --image gcr.io/kpt-fn/kubeval:v0.1 \
 # chaining functions using the unix pipe to set namespace and set labels on
 # wordpress package
 $ kpt fn source wordpress \
-  | kpt fn eval --image gcr.io/kpt-fn/set-namespace:v0.1 - -- namespace=mywordpress \
-  | kpt fn eval --image gcr.io/kpt-fn/set-labels:v0.1 - -- label_name=color label_value=orange \
+  | kpt fn eval - --image gcr.io/kpt-fn/set-namespace:v0.1 - -- namespace=mywordpress \
+  | kpt fn eval - --image gcr.io/kpt-fn/set-labels:v0.1 - -- label_name=color label_value=orange \
   | kpt fn sink wordpress
 ```
 
+```shell
+# execute container 'set-namespace' on the resources in current directory and write
+# the output resources to another directory
+$ kpt fn eval -o path/to/dir --image gcr.io/kpt-fn/set-namespace:v0.1 -- namespace=mywordpress
+```
+
+```shell
+# execute container 'set-namespace' on the resources in current directory and write
+# the output resources to stdout which can be piped to 'kubectl apply'
+$ kpt fn eval -o unwrap --image gcr.io/kpt-fn/set-namespace:v0.1 -- namespace=mywordpress \
+| kubectl apply -
+```
+
+```shell
+# execute container 'set-namespace' on the resources in current directory and write
+# the wrapped output resources to stdout which can be passed to 'set-annotations' function
+# and the final output resources can be written to another directory
```

Comment:
"final output can be written". Describe what the examples is actually doing, not hypotheticals.

Reply:
Updated.